test/tests.js

```diff
@@ -0,0 +1,5 @@
+require('angular');
+require('angular-mocks');
+
+var testsContext = require.context(".", true, /-test$/);
+testsContext.keys().forEach(testsContext);
```

Comment:
Missing new line at EOF.

I think this could also be ES6, because karma is configured to use webpack as a preprocessor.